### PR TITLE
Minor fixes to earlier pages and first attempt at adding hover and question mark text.

### PR DESCRIPTION
**Pull request by TarjinderSingh**

### CODE REVIEW

File: src/browsers/schema/content/homepage.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ The Schizophrenia Exome Sequencing Meta-analysis (SCHEMA) consortium is a large
...
+We would like to thank the many tens of thousands of patients and families who generously contributed to our effort. This project is made possible by the generosity of many funders, including the [Stanley Family Foundation](https://www.broadinstitute.org/stanley), [National Institutes of Mental Health (NIMH)](https://www.nimh.nih.gov/index.shtml), [National Human Genome Research Institute (NHGRI)](https://www.genome.gov/), [Kent and Elizabeth Dauten Foundation](https://dautenbipolarcenter.org/about/dauten-family), and the [Dalio Foundation](https://www.daliophilanthropies.org/initiatives/mental-health-and-wellness/). The [contributing groups](about.md) and the [terms of use](terms.md) are listed on separate pages.
```
Comment (nawatts) [inline src/browsers/schema/content/homepage.md:7]:
```suggestion
We would like to thank the many tens of thousands of patients and families who generously contributed to our effort. This project is made possible by the generosity of many funders, including the [Stanley Family Foundation](https://www.broadinstitute.org/stanley), [National Institutes of Mental Health (NIMH)](https://www.nimh.nih.gov/index.shtml), [National Human Genome Research Institute (NHGRI)](https://www.genome.gov/), [Kent and Elizabeth Dauten Foundation](https://dautenbipolarcenter.org/about/dauten-family), and the [Dalio Foundation](https://www.daliophilanthropies.org/initiatives/mental-health-and-wellness/). The [contributing groups](/about) and the [terms of use](/terms) are listed on separate pages.
```

File: src/browsers/schema/content/generesults.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+PTVs and MPC > 3 missense variants (defined as Class I variants) were jointly analyzed in a single case-control burden test. MPC 2 - 3 variants (defined as Class II variants) were analyzed in a separate burden test before meta-analyzed with the Class I burden P-value using Stouffer's weighted Z-score method. Case-control significance was evaluated using a permutation-based Fisher's Exact Test. For genes with case-control P-value < 0.01, _de novo_ Class I and II P values were calculated using the Poisson rate test and meta-analyzed with our case-control test statistic using a Stouffer's weighted Z-score method. The Q-value is the P-value adjusted for the False Discovery Rate.
+
+For a full description and justification of the approach, please consider the main text and supplementary materials and methods of the SCHEMA pre-print manuscript.  
+
+FIX: space between De novo and P-value
```
Comment (nawatts) [inline src/browsers/schema/content/generesults.md:11]: Done in ccae47fa2ed23f602865df438978ac3774b4d477, these notes can be removed.

File: src/browsers/schema/content/results.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Hover:
```
Comment (nawatts) [inline src/browsers/schema/content/results.md:1]: Tooltips added in ccae47fa2ed23f602865df438978ac3774b4d477. This file can be deleted.

**Review by nawatts — CHANGES_REQUESTED** [review]

File: src/browsers/schema/content/generesults.md (hunk 2 above)
Comment (TarjinderSingh) [reply src/browsers/schema/content/generesults.md]: Removed notes.

File: src/browsers/schema/content/results.md (hunk 3 above)
Comment (TarjinderSingh) [reply src/browsers/schema/content/results.md]: Deleted results.md

**Review by nawatts — APPROVED** [review]

**Pull request merged**